.jitpack.yml

```diff
@@ -1,4 +1,4 @@
 jdk:
- - openjdk11
+ - openjdk18
```

Comment:
stripe-java now requires JDK18 to build:
https://github.com/stripe/stripe-java/pull/1331